ios/SmartSyncExplorerReactNative.xcodeproj/project.pbxproj

```diff
@@ -267,6 +400,7 @@
 				CODE_SIGN_ENTITLEMENTS = SmartSyncExplorerReactNative/SmartSyncExplorerReactNative.entitlements;
 				CURRENT_PROJECT_VERSION = 1;
 				DEAD_CODE_STRIPPING = NO;
+				DEVELOPMENT_TEAM = "";
```

Comment:
You might have to manually remove the `DEVELOPMENT_TEAM` line with a text editor.